metadata.json

```diff
@@ -60,7 +60,7 @@
     },
     {
       "name": "puppet/archive",
-      "version_requirement": ">= 0.3.0 <1.0.0"
+      "version_requirement": ">= 0.3.0 <=1.1.1"
```

Comment:
This was probably a bit tight.  The current version of `puppet/archive` is now `1.1.2`.  I'll open a new PR to address this.